<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1128</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>1071</width>
      <height>621</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_0">
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>762</x>
        <y>157</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>530</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QTableView" name="affichage">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>691</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_supprimer">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>560</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>220</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Trie_botton">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>330</y>
        <width>101</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Tri_nom">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>330</y>
        <width>101</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri NOM</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Tri_adresse">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>330</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri ADRESSE</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rechercheID">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>460</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="rechercheNom">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>460</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="rechercheAdresse">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>460</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>430</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>rechercheID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>430</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>rechercheNom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>430</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>rechercheAdresse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>762</x>
        <y>90</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QQuickWidget" name="quickWidget">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>490</y>
        <width>361</width>
        <height>121</height>
       </rect>
      </property>
      <property name="resizeMode">
       <enum>QQuickWidget::SizeRootObjectToView</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>450</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>MAP</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_1">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>50</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ID_entreprise</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Num_fixe</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>140</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom_entreprise</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>190</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Adress</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>240</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Adressmail</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Valider">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>300</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_id">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_num">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_nom">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_adress">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>190</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_adressmail">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>240</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>85</x>
        <y>150</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom_entreprise</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_id_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>200</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Adress</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_num_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_adress_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>200</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_nom_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>150</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>85</x>
        <y>60</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ID_entreprise</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Modifier">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>310</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_adressmail_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>250</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>250</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Adressmail</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Num_fixe</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_42">
      <property name="geometry">
       <rect>
        <x>301</x>
        <y>51</y>
        <width>75</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Smtp-server:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="port">
      <property name="geometry">
       <rect>
        <x>382</x>
        <y>86</y>
        <width>188</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>465</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="paswd">
      <property name="geometry">
       <rect>
        <x>382</x>
        <y>156</y>
        <width>188</width>
        <height>26</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_55">
      <property name="geometry">
       <rect>
        <x>301</x>
        <y>261</y>
        <width>54</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Message:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rcpt">
      <property name="geometry">
       <rect>
        <x>382</x>
        <y>191</y>
        <width>188</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_48">
      <property name="geometry">
       <rect>
        <x>301</x>
        <y>121</y>
        <width>63</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_53">
      <property name="geometry">
       <rect>
        <x>301</x>
        <y>191</y>
        <width>70</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Recipant to:</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="msg">
      <property name="geometry">
       <rect>
        <x>382</x>
        <y>261</y>
        <width>188</width>
        <height>85</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="subject">
      <property name="geometry">
       <rect>
        <x>382</x>
        <y>226</y>
        <width>188</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="server">
      <property name="geometry">
       <rect>
        <x>382</x>
        <y>51</y>
        <width>188</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>smtp.gmail.com</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_49">
      <property name="geometry">
       <rect>
        <x>301</x>
        <y>156</y>
        <width>59</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sendBtn">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>360</y>
        <width>92</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_54">
      <property name="geometry">
       <rect>
        <x>301</x>
        <y>226</y>
        <width>48</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Subject:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exitBtn">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>360</y>
        <width>92</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="uname">
      <property name="geometry">
       <rect>
        <x>382</x>
        <y>121</y>
        <width>188</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_47">
      <property name="geometry">
       <rect>
        <x>301</x>
        <y>86</y>
        <width>69</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Server port:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1128</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
